database-mongo/index.js

```diff
@@ -13,22 +13,83 @@ db.once('open', function() {
 });
 
 var games = mongoose.Schema({
-  token: String,
-  players: String,
-  results: String  
+  token: {type: String, required: true},	
+  playerRoles: String,
+  results: String, 
+  playerIds: String,
 });
```

Comment:
It doesn't have to be here, but we should have some comments or documentation somewhere that tell us how playerIds, etc are represented. (For the case: real name, client id, others? Related how? Array with set slots? Something else?]